packages/orchestration/src/examples/basic-flows.flows.js

```diff
@@ -103,8 +106,8 @@ export const sendICQQuery = async (orch, _ctx, seat, { chainName, msgs }) => {
     await orch.getChain(chainName)
   );
   const queryResponse = await remoteChain.query(msgs);
-  console.debug('sendICQQuery response:', queryResponse);
-  return queryResponse;
+  trace('SendICQQuery response:', queryResponse);
+  return JSON.stringify(queryResponse);
```

Comment:
I wouldn't say the un-stringified result is incompatible with the smart wallet. It just doesn't get published to vstorage.

suggest:
 - feat: ensure offerHandler result is visible via smart-wallet vstorage